edi_oca/migrations/14.0.1.9.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Téo Goddet
+
+from openupgradelib import openupgrade
+
+_column_renames = {
+    "edi_exchange_record": [("exchange_file", "real_exchange_file")],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.column_exists(
+        env.cr, "openupgrade.rename_columns", "real_exchange_file"
```

Comment:
Why do you need to rename if it is an attachment field ???

Reply:
Well, in any case, this script has no sense openupgade.rename_columns is not a table...

Also, I don't understand why he is creating this new field, some context might clarify things

Reply:
My first migration, sorry for the bad quality, 
I still think that this is needed

the new field is created as requested by simahawk, it allow the exchange_file to be computed from the parent or expliciteky set on the record. If you see a better way to do this, tell me :)